client/app/auth/auth.js

```diff
@@ -0,0 +1,6 @@
+angular.module('throwcast.auth', [])
+
```

Comment:
Let's be consistent with the spacing after the module creation and the chained calls.